File: 2502-sort-the-people/sort-the-people.py
class Solution:
    def sortPeople(self, names: List[str], heights: List[int]) -> List[str]:
        ans = []
        store ={}
        for i in range(len(names)):
            store[heights[i]] = names[i]
        heights.sort(reverse = True)

        for i in range(len(names)):
            ans.append(store[heights[i]])
        return ans

File: 2502-sort-the-people/sort-the-people.cpp
class Solution {
public:
    vector<string> sortPeople(vector<string>& names, vector<int>& heights) {
        unordered_map<int,string> store;
        for(int i=0;i<names.size();i++) store[heights[i]] = names[i];
        sort(heights.rbegin(),heights.rend());
        vector<string> ans;
        for(int i=0;i<names.size();i++) ans.push_back(store[heights[i]]);
        return ans;
    }
};